Loop/Managers/LinearRampDosingStrategy.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  LinearRampDosingStrategy.swift
+//  Loop
+//
+//  Created by Jonas Björkert on 2023-06-03.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKit
+import LoopCore
+
+struct LinearRampDosingStrategy: DosingStrategy {
```

Comment:
"Linear Ramp" is more a detail about how it's doing the calculation. The fact that it's adjusting the application based on glucose is a more primary feature of the strategy. Maybe "GlucoseBasedApplicationFactor" would be better.